automates/text_reading/src/main/resources/org/clulab/aske_automates/grammars/entities/grammar/avoid.yml

```diff
@@ -92,14 +92,18 @@ rules:
     priority: 2
     type: token
     pattern: |
-      [entity=/PERSON|ORGANIZATION|LOCATION/]{2,3}
+      [entity=/PERSON|ORGANIZATION|LOCATION/]{2,4} #note: adjusted {2,3} -> {2,4} to account for more cases. Can we adjust the number to {3,4}? (i.e. Mainland China, Hong Kong not captured as a phrase due to this rule)
```

Comment:
I ran the tests after adjusting the number from {2,4} to {3,4} and removing Loc, and it didn't break any passing test. So I think it'll be safe enough to revise the rule as `[entity=/PERSON|ORGANIZATION/]{3,4}`.

Reply:
I now seem to remember that this was introduced because there were a lot of two part personal names were extracted as variables, so if it is not interfering with what you want done, I'd have it as [entity=/PERSON|ORGANIZATION/]{2,4}

Reply:
Oh, if that's the case, then we should keep the rule as you suggested. Let's keep the number as {2,4}.